src/core/client/admin/components/StoryInfoDrawer/ModerateStoryButton.tsx

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable */
+import { Localized } from "@fluent/react/compat";
+import React, { FunctionComponent, useMemo } from "react";
+import { graphql } from "react-relay";
+
+import { Button } from "coral-ui/components/v3/Button/Button";
+import { getModerationLink } from "coral-framework/helpers";
+import { withFragmentContainer, useLocal } from "coral-framework/lib/relay";
+import { Ability, can } from "coral-framework/permissions";
+
+import { ModerateStoryButton_local } from "coral-admin/__generated__/ModerateStoryButton_local.graphql";
+import { ModerateStoryButton_settings } from "coral-admin/__generated__/ModerateStoryButton_settings.graphql";
+import { ModerateStoryButton_story } from "coral-admin/__generated__/ModerateStoryButton_story.graphql";
+import { ModerateStoryButton_viewer } from "coral-admin/__generated__/ModerateStoryButton_viewer.graphql";
+
+import * as styles from "./ModerateStoryButton.css";
+
+export interface Props {
+  settings: ModerateStoryButton_settings;
+  viewer: ModerateStoryButton_viewer;
+  story: ModerateStoryButton_story;
+}
+
+const ModerateStoryButton: FunctionComponent<Props> = ({
+  settings,
+  story: { id, isArchived, isArchiving, canModerate },
+  viewer
+}) => {
+  const [{ accessToken }] = useLocal<ModerateStoryButton_local>(graphql`
+    fragment ModerateStoryButton_local on Local {
+      accessToken
+    }
+  `);
+
+  const href = useMemo(() => {
+    let link = getModerationLink({ storyID: id });
+    if (
+      accessToken &&
+      settings.auth.integrations.sso.enabled &&
+      settings.auth.integrations.sso.targetFilter.admin
+    ) {
+      link += `#accessToken=${accessToken}`;
+    }
+
+    return link;
+  }, [accessToken, settings, id]);
+
+  if (!canModerate || !viewer || !can(viewer, Ability.MODERATE)) {
+    return null;
+  }
+
+  if (isArchived || isArchiving) {
+    return null;
+  }
+
+  return (
+    <Localized id="storyInfoDrawer-moderateStory">
+      <Button
+        className={styles.moderateStory}
+        variant="outlined"
+        color="primary"
+        paddingSize="small"
+        fontSize="extraSmall"
+        href={href}
+        target="_blank"
+        upperCase
+      >
+        Moderate
```

Comment:
In the localization file it says `Moderate story`, and here and in the stream it just says `Moderate`. Which do we want for this button?